src/js/containers/covid19/recipient/SpendingByRecipientContainer.jsx

```diff
@@ -280,15 +280,15 @@ const SpendingByRecipientContainer = ({ activeTab, scrollIntoView }) => {
         else {
             changeCurrentPage(1);
         }
-    }, [pageSize, defCodes, sort, order, activeTab, query]);
+    }, [pageSize, defCodes, sort, order, activeTab, query, currentPage, fetchSpendingByRecipientCallback]);
```

Comment:
I think one of these dependency array changes is causing the Recipient table to crash. I'm seeing 
```
react_devtools_backend.js:2430 Warning: Maximum update depth exceeded. This can happen when a component calls setState inside useEffect, but useEffect either doesn't have a dependency array, or one of the dependencies changes on every render.
    in SpendingByRecipientContainer (created by SpendingByRecipient)
```
and then the browser stops responding